src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--active .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
```

Comment:
Add code for other modifiers like 1 through 5

Reply:
Added.